modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BoundObjectDefinitionsExportImportTest.java

```diff
@@ -0,0 +1,280 @@
+				objectDefinitionJSONObject.getJSONArray("objectFields");
+
+			objectFieldsJSONArray.put(
+				_jsonFactory.createJSONObject(
+				).put(
+					"name",
+					StringBundler.concat(
+						"r_objectRelationship", i, "_c_testObjectDefinition", i,
+						"Id")
+				));
+		}
+
+		testExportImportJSONString(
+			draftBoundObjectDefinitionsJSONString,
+			expectedDraftBoundObjectDefinitionsJSONArray.toString(), null,
+			"TestObjectDefinition1");
+
+		JSONArray publishedBoundObjectDefinitionsJSONArray =
+			_jsonFactory.createJSONArray(draftBoundObjectDefinitionsJSONString);
+
+		for (Object publishedBoundObjectDefinition :
+				publishedBoundObjectDefinitionsJSONArray) {
+
+			JSONObject objectDefinitionJSONObject =
+				(JSONObject)publishedBoundObjectDefinition;
+
+			objectDefinitionJSONObject.put(
+				"status",
+				_jsonFactory.createJSONObject(
+				).put(
+					"code", 0
+				).put(
+					"label", "approved"
+				).put(
+					"label_i18n", "Approved"
+				));
+		}
+
+		testExportImportJSONString(
+			publishedBoundObjectDefinitionsJSONArray.toString(),
+			expectedDraftBoundObjectDefinitionsJSONArray.toString(), null,
+			"TestObjectDefinition1");
+
+		JSONArray invalidBoundObjectDefinitionsJSONArray =
+			_jsonFactory.createJSONArray(draftBoundObjectDefinitionsJSONString);
+
+		for (Object invalidBoundObjectDefinition :
+				invalidBoundObjectDefinitionsJSONArray) {
+
+			JSONObject objectDefinitionJSONObject =
+				(JSONObject)invalidBoundObjectDefinition;
+
+			objectDefinitionJSONObject.put(
+				"name", "!@" + objectDefinitionJSONObject.get("name"));
+		}
+
+		Class<?> clazz = getClass();
+
+		testFailedImportJSONString(
+			invalidBoundObjectDefinitionsJSONArray.toString(),
+			StringUtil.read(
+				clazz.getResourceAsStream(
+					"dependencies/test-invalid-bound-object-definitions." +
+						"error-message.json")),
+			null, null);
+	}
+
+	@Override
+	protected ClassLoader getClassLoader() {
+		return BoundObjectDefinitionsExportImportTest.class.getClassLoader();
+	}
+
+	@Override
+	protected Class<?> getClazz() {
+		return getClass();
+	}
+
+	@Override
+	protected long getId(String name) throws Exception {
+		Page<ObjectDefinition> objectDefinitionsPage =
+			_objectDefinitionResource.getObjectDefinitionsPage(
+				name, null, null, Pagination.of(1, 1), null);
+
+		List<ObjectDefinition> objectDefinitions =
+			(List<ObjectDefinition>)objectDefinitionsPage.getItems();
+
+		ObjectDefinition objectDefinition = objectDefinitions.get(0);
+
+		return objectDefinition.getId();
+	}
+
+	@Override
+	protected String getIdentifierName() {
+		return "objectDefinitionId";
+	}
+
+	@Override
+	protected String getJSONName() {
+		return "objectDefinitionJSON";
+	}
+
+	@Override
+	protected MVCActionCommand getMVCActionCommand() {
+		return _mvcActionCommand;
+	}
+
+	@Override
+	protected MVCResourceCommand getMVCResourceCommand() {
+		return _mvcResourceCommand;
+	}
+
+	private JSONObject _createOneToManyObjectRelationship(
```

Comment:
Can you move this method to the base class? 
This way, `ObjectDefinitionExportImportTest` can reuse this method too.